src/Rules/Functions/PrintfParametersRule.php

```diff
@@ -109,6 +112,17 @@ public function processNode(Node $node, Scope $scope): array
 
 	private function getPlaceholdersCount(string $functionName, string $format): int
 	{
+		try {
```

Comment:
instead of re-implemeting the complex php-src argument parsing, I just delegate to the runtime and interpret the error message.

this works as of php8+ when php-src started throwing `ArgumentCountError`